File: examples/php-fixture/tests/CalculatorTest.php
<?php

declare(strict_types=1);

namespace Example\Tests;

use Example\Calculator;
use PHPUnit\Framework\TestCase;

final class CalculatorTest extends TestCase
{
    public function testAdd(): void
    {
        self::assertSame(5, (new Calculator())->add(2, 3));
    }

    public function testMultiply(): void
    {
        self::assertSame(6, (new Calculator())->multiply(2, 3));
    }
}

File: examples/ruby-fixture/test/calculator_test.rb
require "minitest/autorun"
require "calculator"

class CalculatorTest < Minitest::Test
  def test_add
    assert_equal 5, Calculator.add(2, 3)
  end

  def test_multiply
    assert_equal 6, Calculator.multiply(2, 3)
  end
end
